### DISCUSSION TITLE
Are GPTQ's group-size, --true-sequential and act-order supported by llama.cpp?

### DISCUSSION THREAD

**Discussion by x02Sylvie** [ggml-org/llama.cpp#580] [General]
As mentioned here https://github.com/qwopqwop200/GPTQ-for-LLaMa

These three can greatly improve results, so im wondering

**Comment by BadisG** [comment]
Groupsize yeah https://github.com/ggerganov/llama.cpp/pull/545
For the two other we haven't tested it yet, someone has to convert the HF weights into .pt with those two parameters first, I can't I don't have enough vram :'(
